sdcm/cluster.py

```diff
@@ -5390,12 +5390,19 @@ def install_scylla_monitoring_prereqs(self, node):  # pylint: disable=invalid-na
 
         if node.distro.is_rhel_like:
             node.install_epel()
-            node.install_package(package_name="unzip wget python36 python36-pip")
-            prereqs_script = dedent("""
-                python3 -m pip install --upgrade pip
-                python3 -m pip install pyyaml
+            node.install_package(package_name="unzip wget python3 python3-pip")
+            # get-docker.sh doesn't support Rocky, thus docker should be installed from repo via dnf install
+            install_docker_cmd = dedent("""
+                dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+                dnf install -y docker-ce docker-ce-cli containerd.io
+            """ if node.distro.is_rocky else """
                 curl -fsSL get.docker.com --retry 5 --retry-max-time 300 -o get-docker.sh
                 sh get-docker.sh
+            """)
+            prereqs_script = dedent(f"""
+                python3 -m pip install --upgrade pip
+                python3 -m pip install pyyaml
+                {install_docker_cmd}
```

Comment:
`pip`-specific actions are not required for non-rocky distr.
So, need to make it be conditioned to avoid unnecessary breakages if pip code gets broken for any reason.